src/app/components/card/sass/_base.scss

```diff
@@ -0,0 +1,17 @@
+html {
+  font-size: 62.5%;
+
+  @include respond(tab-land) {
+    // width < 1200?
+    font-size: 56.25%; //1 rem = 9px, 9/16 = 50%
+  }
+
+  @include respond(tab-port) {
+    // width < 900?
+    font-size: 50%; //1 rem = 8px, 8/16 = 50%
```

Comment:
Hey @Jaman-dedy, these comments are not very helpful, we should consider cleaning them up

Reply:
Thanks @CryceTruly , I worked on that